src/arch/riscv/kernel/vspace.c

```diff
@@ -207,10 +207,10 @@ BOOT_CODE void map_it_frame_cap(cap_t vspace_cap, cap_t frame_cap)
     *targetSlot = pte_new(
                       (pptr_to_paddr(frame_pptr) >> seL4_PageBits),
                       0, /* sw */
-                      1, /* dirty */
-                      1, /* accessed */
+                      1, /* dirty leaf/reserved non-leaf */
+                      1, /* accessed leaf/reserved non-leaf */
```

Comment:
Does this not have to be 0 for non-leaf?

Reply:
I decided to make all of the comments the same, though I can change that if desired. Let me know.

Reply:
It confused me in this one, so it's probably better to change. Maybe `/* accessed (leaf) */` would work, to indicate that we haven't forgotten about the leaf/no-leaf distinction and know which case we are in.